bot_modules/server_status_checker/bedrockProtocolVersion.js

```diff
@@ -0,0 +1,31 @@
+const Utils = require('../../utils')
+
+const xpath = require('xpath')
+const parse5 = require('parse5')
+const xmlser = require('xmlserializer')
+const Dom = require('xmldom').DOMParser
+
+let wikiDoc
+let xPathSelect
+
+async function init () {
+  // https://stackoverflow.com/questions/25753368/performant-parsing-of-pages-with-node-js-and-xpath
+
+  const html = (await Utils.getContents('https://minecraft.gamepedia.com/Template:Protocol_version/Table/doc')).data
+  const document = parse5.parse(html.toString())
+  const xhtml = xmlser.serializeToString(document)
+  wikiDoc = new Dom().parseFromString(xhtml)
+  xPathSelect = xpath.useNamespaces({ x: 'http://www.w3.org/1999/xhtml' })
```

Comment:
Could this be replaced with cheerio? (To save the 4 extra dependencies)

Reply:
I'll see if can